platform-dom/src/main/java/com/softicar/platform/dom/engine/IDomEngine.java

```diff
@@ -102,10 +104,60 @@ public interface IDomEngine {
 	 */
 	void setFireOnKeyUp(IDomNode node, DomEventType type, boolean enabled);
 
+	/**
+	 * Tells the client browser to not perform the default action if an event of
+	 * the given {@link DomEventType} occurs on the given {@link IDomNode}.
```

Comment:
I think this only works for key events, we should list which DomEventType instances that are.